networkx/classes/tests/test_reportviews.py

```diff
@@ -873,7 +873,7 @@ def test_iterkeys(self):
         elist = sorted([(i, i + 1, 0) for i in range(8)] + [(1, 2, 3)])
         assert sorted(list(ev)) == elist
         # test order of arguments:graph, nbunch, data, keys, default
-        ev = evr((1, 2), "foo", True, 1)
+        ev = evr((1, 2), "foo", keys=True, default=1)
```

Comment:
Should we add a test that it is keyword only?  I mean something like:
```python
with pytest.raises(TypeError):
    evr((1, 2), "foo", True, 1)
```
Or is it overkill to test for whether the arguments are keyword only or not.  I guess to do a thorough job you'd want to test that using a second positional argument doesn't raise a TypeError, while the third positional argument does. If we're going to be eventually using keyword-only arguments in many places in the library the complete tests could become onerous.  But it would have caught the potential bug that this PR fixes...

Reply:
Thank you, @dschult ! I implemented this, but I agree that this won't scale well (in general) if keyword-only arguments are more widely used.